app/src/main/java/fr/neamar/kiss/notification/NotificationListener.java

```diff
@@ -149,7 +160,7 @@ public void onNotificationRemoved(StatusBarNotification sbn) {
 
     /**
      * Check for trivial notifications.
-     *
+     * <p>
```

Comment:
A typo?

Reply:
Reverted, this was introduced because of Android Studios "Generate "<p/>" on empty lines" setting.